examples/dnsdoctor/setup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -eux
+
+# add namespaces ns1 for dns server ns2 for server
```

Comment:
why do we need namespaces for this example? btw, I would change `resolv.conf` as well.. I would just `dig` to a well known domain (e.g., `ebpf.io`) and "redirect" it later to a localhost..

Reply:
I wanted to create the DNS doctoring scenario using namespaces.